### ISSUE TITLE
Constant jitter spikes causing mistimed and delayed frame using display-resample

### ISSUE THREAD

**Issue by BluTree** [mpv-player/mpv#11791]
### Important Information

Provide following Information:
- mpv version :  0.35.0-436-g1c82d6ae
- Windows Version : W11 22621.1778
- Source of the mpv binary : https://sourceforge.net/projects/mpv-player-windows/

### Reproduction steps

launch mpv with the f

`mpv --no-config --video-sync=display-resample`

Try to reproduce your issue with --no-config first. If it isn't reproducible
with --no-config try to first find out which option or script causes your issue.

Describe the reproduction steps as precise as possible. It's very likely that
the bug you experience wasn't reproduced by the developer because the workflow
differs from your own.

### Expected behavior

Having no mistimed nor delayed frames

### Actual behavior

Constant jitter spikes that causes mistimed and delayed frames

### Log file

Make a log file made with -v -v or --log-file=output.txt, paste it to
https://0x0.st/ or attach it to the github issue, and replace this text with a
link to it.

The issue will be closed for ignoring the issue template.

### Sample files

Sample files needed to reproduce this issue can be uploaded to https://0x0.st/
or similar sites. (Only needed if the issue cannot be reproduced without it.)
Do not use garbage like "cloud storage", especially not Google Drive.

**Comment by Dudemanguy** [comment]
Do you have gsync or freesync enabled by any chance?

**Comment by BluTree** [comment]
No, it is disabled on my monitor directly, so Nvidia doesn't even recognize my monitor as a gsyn compatible monitor.

I also saw a similar issue talking about high frame time (#11863), and tried changing the hwdec as well as gpu-api.
I am currently with `hwdec=d3d11va-copy` and `gpu-api=vulkan`, and I never encountered this kind of jitter again.
Still, this is a very strange behaviour.

**Comment by Dudemanguy** [comment]
It has to be a NVIDIA driver issue in that case. There's no reason why syncing to vblank should jump around so much like that.

**Comment by mia-0** [comment]
I’ve observed behavior like this on Linux years ago. In my case, the GPU load caused by mpv hovered around a power management threshold so NVIDIA’s driver was constantly switching between power states, which looked very similar to what we can see in the video. If forcing performance mode in the driver settings resolves this issue, it’s most likely the same problem. If so, that’s probably not fixable on our side.

**Comment by BluTree** [comment]
I already tried this, as well as setting mpv as "High Performance" in the W11 settings, but the problem still appears.
